packages/core/src/services/ai/helpers.ts

```diff
@@ -66,8 +66,8 @@ export function createProvider({
 export function validateConfig(config: Record<string, unknown>): Config {
   const configSchema = z
     .object({
-      model: z.string(),
-      provider: z.string(),
+      model: z.string().optional(), // TODO: make it not optional if we ever remove the default provider
+      provider: z.string().optional(), // TODO: make it not optional if we ever remove the default provider
```

Comment:
This is not a todo, maybe a conditional todo xD

Reply:
good point